src/lib.rs

```diff
@@ -19,7 +19,7 @@
 //!
 //! - [General `GLib` family types and object system overview](../glib/index.html)
 //!
-//! - [GTK+ documentation](http://www.gtk.org/documentation.php)
+//! - [GTK+ documentation](https://www.gtk.org/docs/)
```

Comment:
Maybe should also drop the `+` from the name since it got renamed to just GTK. Best be done in a separate MR/issue as its probably many more places.